recipes/fenics-oasis/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "fenics-oasis" %}
+{% set version = "2019.1" %}
+{% set python_min = "3.7" %}
```

Comment:
This is not supported on conda-forge. While we can merge we won't guarantee it will work on py<39

Reply:
@ocefpaf I have updated the minimum Python version to 3.9 as suggested. Thanks!